temporian/core/data/BUILD

```diff
@@ -43,3 +43,10 @@ py_library(
         ":sampling",
     ],
 )
+
```

Comment:
Should you add duration dependency on sma core ?

```
py_library(
    name = "simple_moving_average",
    srcs = ["simple_moving_average.py"],
    srcs_version = "PY3",
    deps = [
        ":base",
        "//temporian/core:operator_lib",
        "//temporian/core/data:duration", --> NEW
        "//temporian/core/data:event",
        "//temporian/core/data:feature",
        "//temporian/proto:core_py_proto",
    ],
)
```

Reply:
Good catch.
*Solved.

Note: In practice, python+bazel seems to allow for indirect dependencies i.e. this code compiled because on the of dependencies of :simple_moving_average was depending on :duration.